libindy-crypto/src/bn/openssl.rs

```diff
@@ -414,7 +414,7 @@ impl BigNumber {
             sha256.update(&num)?;
         }
 
-        Ok(sha256.finish2()?.to_vec())
+        Ok(sha256.finish()?.to_vec())
```

Comment:
I hope it's correct that `finish` in the new API corresponds to `finish2` in the old API.